src/core/actor/DipolarBarnesHut_cuda.cu

```diff
@@ -250,7 +250,7 @@ __global__ __launch_bounds__(THREADS2, FACTOR2) void treeBuildingKernel() {
       j = 0;
       for (l = 0; l < 3; l++)
         if (root[l] < p[l])
-          j += pow(2, l);
+          j += static_cast<int>(pow(2, l));
```

Comment:
@RudolfWeeber if you referred to these integer powers in the dipolar code, `int_pow<l>()` will not work since the power is not constexpr. Could be a candidate for bit shifting, though, but I don't think this is performance critical because there is an `if()` right above it, which is not a good idea in a CUDA kernel.